cobalt/h5vcc/h5vcc_settings.cc

```diff
@@ -118,6 +118,15 @@ bool H5vccSettings::Set(const std::string& name, SetValueType value) const {
     }
   }
 
+  if (name.compare(network::kProtocolFilterKey) == 0 &&
+      value.IsType<std::string>() && value.AsType<std::string>().size() < 512) {
```

Comment:
Oh, I think a max length of 512 is too short. Let's set it nice and high, like 16384 or so.